recipes/stim/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: stim
+  version: {{ version }}
+
+source:
+  url: https://github.com/PreibischLab/STIM/archive/refs/tags/v0.3.0-alpha.zip
```

The versions do not match. You should use a jinja template for the version in the URL. Last but not least, dev release should follow CFEP-05: https://github.com/conda-forge/cfep/blob/main/cfep-05.md 

```suggestion
{% set version = "0.2.0" %}

package:
  name: stim
  version: {{ version }}

source:
  url: https://github.com/PreibischLab/STIM/archive/refs/tags/v0.3.0-alpha.zip
```